scene/main/node.cpp

```diff
@@ -1892,7 +1938,7 @@ bool Node::is_unique_name_in_owner() const {
 	return data.unique_name_in_owner;
 }
 
-void Node::set_owner(Node *p_owner) {
+void Node::_set_owner(Node *p_owner, bool p_show_warning) {
```

Comment:
What is this change for? Don't you already ensure this when checking the owner beforehand?

Reply:
you are totally correct - i think i just kept the old implementation and didn't think about it. i'll go ahead and get this cleaned up too